docs/pyretriever.rst

```diff
@@ -52,8 +52,25 @@ The function returns a list of all the currently available scripts.
 
 
 For more detailed description of the scripts installed in retriever ``datasets`` function can be used.
+
```

Comment:
lets remove the example because we are printing out objects. 
or we can say 
```
script = [dataset for dataset in rt.datasets()]
script[0].name
```

I prefer not showing the example until the second part when we print out names